src/mammos_mumag/hysteresis.py

```diff
@@ -109,11 +109,37 @@ def run(
         ),
     )
     sim.run_loop(outdir=outdir, name="hystloop")
-    df = pd.read_csv(
-        f"{outdir}/hystloop.dat",
-        delimiter=" ",
-        names=["configuration_type", "mu0_Hext", "polarisation", "energy_density"],
-    )
+    return read_result(outdir=outdir, name="hystloop")
+
+
+def read_result(
+    outdir: pathlib.Path,
+    name: str = "out",
+) -> Result:
+    r"""Read hysteresis loop output from directory.
+
+    Args:
+        outdir: Path of output directory where the results of the hysteresis loop are
+            stored.
+        name: System name with which the loop output files are stored.
+
+    Returns:
+       Result object.
+
+    Raises:
+        FileNotFoundError: hysteresis loop .dat file not found.
+
+    """
+    try:
+        df = pd.read_csv(
+            f"{outdir}/{name}.dat",
```

Comment:
Please change this to ensure it also works on Windows where the separator is different.

```suggestion
            outdir / f"{name}.dat",
```

(If you accept the other suggestion `Path(outdir) / f"{name}.dat"`)